### PR TITLE
unified naming convention

### PR DESCRIPTION
**Pull request by wbpcode**
Signed-off-by: wbpcode <comems@msn.com>

* Unified naming convention.
* Keep naming convension same with [envoy](https://github.com/envoyproxy/envoy/blob/main/STYLE.md).

ps: A new pr is needed to update demo.

### CODE REVIEW

File: common/common.h [hunk 1]
```diff
@@ -4,7 +4,7 @@
 #include <sstream>
 #include <string>
 
-namespace hessian2 {
+namespace Hessian2 {
 class utils {
```
Comment (zyfjeff) [inline common/common.h:8]: utils -> Utils?

File: example/custom_type.cc [hunk 2]
```diff
@@ -11,10 +11,10 @@ struct Person {
 // The custom struct needs to implement from_hessian and to_hessian methods to
 // encode and decode
 
-void from_hessian(Person&, ::hessian2::Decoder&);
-bool to_hessian(const Person&, ::hessian2::Encoder&);
+void from_hessian(Person&, ::Hessian2::Decoder&);
```
Comment (zyfjeff) [inline example/custom_type.cc:14]: from_hessian -> fromHessian?

File: example/custom_type.cc [hunk 3]
```diff
@@ -11,10 +11,10 @@ struct Person {
...
+bool to_hessian(const Person&, ::Hessian2::Encoder&);
```
Comment (zyfjeff) [inline example/custom_type.cc:15]: ditto

**Review by zyfjeff — CHANGES_REQUESTED** [review]
Great work, Some small comments.

File: common/common.h (hunk 1 above)
Comment (wbpcode) [reply common/common.h:8]: Get it.

**Comment by zyfjeff** [comment]
Great work! 🎉

**Review by zyfjeff — APPROVED** [review]

**Pull request merged**